action.yml

```diff
@@ -87,6 +87,9 @@ inputs:
   json_file:
     description: 'Results are written to this JSON file.'
     required: false
+  job_summary:
+    description: 'Publish results on job summary page. Can be used in conjunction or replacement to pull request decoration'
+    required: false
```

Comment:
please add the default value here as well

Reply:
There's no default in the model, you mean mention it in description ?

Reply:
My bad, it was just highlighted because of type